cibuildwheel/windows.py

```diff
@@ -43,8 +45,9 @@ def get_python_configurations(build_selector):
         PythonConfiguration(version='3.7.6', arch='64', identifier='cp37-win_amd64', url=None),
         PythonConfiguration(version='3.8.2', arch='32', identifier='cp38-win32', url=None),
         PythonConfiguration(version='3.8.2', arch='64', identifier='cp38-win_amd64', url=None),
-        PythonConfiguration(version='2.7-v7.3.0', arch='32', identifier='pp27-win32', url='https://bitbucket.org/pypy/pypy/downloads/pypy2.7-v7.3.0-win32.zip'),
-        PythonConfiguration(version='3.6-v7.3.0', arch='32', identifier='pp36-win32', url='https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v7.3.0-win32.zip'),
+        # PyPy
+        PythonConfiguration(version='2.7', arch='32', identifier='pp27-win32', url='https://bitbucket.org/pypy/pypy/downloads/pypy2.7-v7.3.0-win32.zip'),
```

Comment:
The removal of the PyPy version might be problematic once there are new PyPy releases? :-/
I.e., it might be useful to know when to apply the current workarounds. But this is internal, so I guess we'll fix that later?

Reply:
There are some problem with always using newest release?

Reply:
Oh, sorry, I see now, in `macos.py`. Still possible, based on the `url` field.

Reply:
> There are some problem with always using newest release?

Maybe. From 7.3 to 7.4, the ABI would be broken. So it might be nice to build wheels for the last couple of ABI tags?